src/Compiler/Microsoft.AspNetCore.Razor.Language/src/LargeTextSourceDocument.cs

```diff
@@ -125,7 +125,7 @@ public override byte[] GetChecksum()
             var bytes = new byte[byteCount];
             encoder.GetBytes(charBuffer, 0, charBuffer.Length, bytes, 0, flush: true);
 
-            using (var hashAlgorithm = SHA1.Create())
+            using (var hashAlgorithm = SHA256.Create())
```

Comment:
Rather than hard coding `SHA256` in a bunch of places we should have a centralized place that every place uses. That way future changes, and audits, are limited to a single location.

Reply:
Do we know of a good centralized place I could refactor this to? I have to check

Reply:
You could put it in `Microsoft.AspNetCore.Razor.Utilities.Shared`. That's shared across the compiler and tooling layers.